workflow/postnord/postnord.go

```diff
@@ -0,0 +1,62 @@
+package postnord
+
+import (
+	"fmt"
+	"go.uber.org/cadence/worker"
+	"go.uber.org/cadence/workflow"
+	"go.uber.org/zap"
+	"time"
+)
+
+const (
+	WorkflowName = "OrderProcessingWorkflow"
+)
+
+// RegisterWorkflow registers the OrderProcessingWorkflow.
+func RegisterWorkflow(w worker.Worker) {
+	w.RegisterWorkflowWithOptions(OrderProcessingWorkflow, workflow.RegisterOptions{Name: WorkflowName})
+	// Register your activities here
+}
+
+// Order represents an order with basic details like the ID, customer name, and order amount.
+type Order struct {
+	ID       string  `json:"id"`
+	Customer string  `json:"customer"`
+	Amount   float64 `json:"amount"`
```

Comment:
amount of goods or money? Can it be fractional?

Reply:
it is an order amount, it refers to the total sum of the prices for all items in an order